### ISSUE TITLE
Compiled release 0.3.2

### ISSUE THREAD

**Issue by Jurollet** [marian-m12l/studio#459]
Hey everyone !
Thanks to you all for all this great work !

**Describe the bug**
Necessary dependencies to building `studio-web-ui` seem not to be possible to download on an Apple M1 Macbook ([see related issue here](https://github.com/eirslett/frontend-maven-plugin/issues/952#issuecomment-760723172)).

Could someone simply release the 0.3.2 version with compiled software, so we can enjoy it even on those devices, for the latest versions of Lunii ? :)

Thanks !

**To Reproduce**
2. `git clone`
3. `mvn install clean`

**Expected behavior**
STUdio is built.

**Logs**
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for studio-parent 0.3.2-SNAPSHOT:
[INFO]
[INFO] studio-parent ...................................... SUCCESS [  0.171 s]
[INFO] studio-metadata .................................... SUCCESS [  1.272 s]
[INFO] studio-agent ....................................... SUCCESS [  0.683 s]
[INFO] studio-core ........................................ SUCCESS [  0.543 s]
[INFO] studio-driver ...................................... SUCCESS [  0.406 s]
[INFO] studio-web-ui ...................................... FAILURE [  2.167 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.319 s
[INFO] Finished at: 2024-06-11T17:07:08+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.7.6:install-node-and-yarn (install-frontend-tools) on project studio-web-ui: Could not download Node.js: Got error code 404 from the server. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.7.6:install-node-and-yarn (install-frontend-tools) on project studio-web-ui: Could not download Node.js: Got error code 404 from the server.
```

**Desktop and environment (please complete the following information):**
 - OS: macOS
 - Java version: JDK 22.0.1
 - Maven version: 3.9.7
 - STUdio application version 0.3.2

**Comment by marian-m12l** [comment]
Hi,

Thanks for the feedback.

Did you try bumping up the plugin version?

Anyway, I should probably make an actual release with support for v3 devices. I was hoping to get more feedback before making a release, since I can't test it on an actual device myself, but it's probably the best way forward now.

**Comment by marian-m12l** [comment]
Released version 0.4.0
